v1/runtime/runtime.go

```diff
@@ -282,6 +282,15 @@ func NewParams() Params {
 	}
 }
 
+type ServerStatus int
```

Comment:
I guess performance is irrelevant here, but I'd still go with something like uint8 unless we expect to have more than 255 states sometime soon :)